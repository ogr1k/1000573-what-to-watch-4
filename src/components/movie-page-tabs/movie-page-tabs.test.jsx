import React from "react";
import renderer from "react-test-renderer";
import MoviePageTabs from "./movie-page-tabs.jsx";
import {Router} from "react-router-dom";
import history from "../../history.js";


it(`Render Movie Page Tabs`, () => {

  const tree = renderer
    .create(

        <Router history={history}>
          <MoviePageTabs
            activeTab={``}
            filmId={1}
          />
        </Router>

    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
