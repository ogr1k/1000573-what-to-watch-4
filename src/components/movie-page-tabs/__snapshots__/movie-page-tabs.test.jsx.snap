// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Movie Page Tabs 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <li
      className="movie-nav__item"
    >
      <a
        className="movie-nav__link"
        href="/films/1"
        onClick={[Function]}
      >
        Overview
      </a>
    </li>
    <li
      className="movie-nav__item"
    >
      <a
        className="movie-nav__link"
        href="/films/1/details"
        onClick={[Function]}
      >
        Details
      </a>
    </li>
    <li
      className="movie-nav__item"
    >
      <a
        className="movie-nav__link"
        href="/films/1/review"
        onClick={[Function]}
      >
        Reviews
      </a>
    </li>
  </ul>
</nav>
`;
